crates/bevy_window/src/event.rs

```diff
@@ -1,25 +1,19 @@
 use std::path::PathBuf;
 
-use super::{WindowDescriptor, WindowId};
+use bevy_ecs::entity::Entity;
 use bevy_math::{IVec2, Vec2};
 
 /// A window event that is sent whenever a window's logical size has changed.
 #[derive(Debug, Clone)]
 pub struct WindowResized {
-    pub id: WindowId,
-    /// The new logical width of the window.
-    pub width: f32,
+    pub entity: Entity,
+    /// The new logical width of the window
+    pub width: f64,
```

Comment:
Is this to match with winit's representation?

Reply:
Ya